Gemfile.lock

```diff
@@ -154,6 +155,7 @@ GEM
     bson (4.15.0)
     builder (3.2.4)
     byebug (11.1.3)
+    chunky_png (1.4.0)
```

Comment:
I'd like to personally see a warning somewhere about not using this to process untrusted input. Maybe we could add this as a comment into this file? Don't know where the best place would be; I imagine here might be best, but informing readers of the code about this will help prevent potential pitfalls down the line.